packages/omg-js-childchain/src/transaction/transaction.js

```diff
@@ -21,6 +21,9 @@ const MAX_OUTPUTS = 2
 const NULL_INPUT = { blknum: 0, txindex: 0, oindex: 0 }
 const NULL_OUTPUT = { owner: '0x0000000000000000000000000000000000000000', amount: 0 }
 
+const BLOCK_OFFSET = 1000000000
```

Comment:
I'm not familiar with this, what is the use of BlOCK_OFFSET and TX_OFFSET ?

Reply:
@Pongch This comes from the RootChain contract [startExit function](https://github.com/omisego/plasma-contracts/blob/3cc6097b11c949371fee8a64d53f468f616883ae/contracts/RootChain.sol#L205)
It's a way of combining `blknum`, `txindex` and `oindex` to identify a utxo

Reply:
@Pongch This comes from the RootChain contract [startExit function](https://github.com/omisego/plasma-contracts/blob/3cc6097b11c949371fee8a64d53f468f616883ae/contracts/RootChain.sol#L205)
It's a way of combining `blknum`, `txindex` and `oindex` to identify a utxo